CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+# Contributing to @prezly/theme-kit-nextjs
+
+First off, thanks for taking the time to contribute! 🎉
+
+Assuming you have cloned the repo and installed dependencies on your machine, the very next step is to develop the package.
+
+Since theme-kit provides a collection of base functions, hooks, and utilities that works with [Prezly themes](https://github.com/prezly?q=theme-nextjs&type=all&language=&sort=) under the hood, in order to make changes and test things out on your side, you're gonna need to keep a theme running in parallel with the theme-kit.
```

Comment:
```suggestion
Since theme-kit provides a collection of base functions, hooks, and utilities that work with [Prezly themes](https://github.com/prezly?q=theme-nextjs&type=all&language=&sort=) under the hood, in order to make changes and test things out on your side, you're gonna need to keep a theme running in parallel with the theme-kit.
```